gnome/virt-manager/files/virt-manager.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 
+PATH=/opt/local/bin:/opt/local/sbin:${PATH:-$(/usr/bin/getconf PATH)}
```

Comment:
Per @pmetzger, you don't want to hard-code the path to `/opt/local`.

But the port is already setup to deal with this, and there's an easy fix: Simply use placeholder `@PREFIX@` instead.

Reply:
It looks like you'll also need to update the `reinplace` statement on line 87, as it's not currently performing a global replace. And this will prevent replacement of multiple occurrences on the same line.

So change the following on that line:

`s+@PREFIX@+${prefix}+`

to:

`"s|@PREFIX@|${prefix}|g"`

Reply:
Thanks for your comment @mascguy. Although @pmetzger's comment made sense to me I was a bit uncertain what I needed to do (I'm new to manipulating Portfiles).

I incorporated your suggestions in bbae678.